packages/stream_video/CHANGELOG.md

```
@@ -23,6 +23,10 @@ See the [documentation](https://getstream.io/video/docs/flutter/guides/camera-an
 ### ⚠️ Deprecated
 * Deprecated `androidAudioConfiguration` in `StreamVideoOptions`. Use `audioConfigurationPolicy` instead.
 
+### 🐞 Fixed
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)